perllib/FixMyStreet/Cobrand/Northumberland.pm

```diff
@@ -292,6 +294,65 @@ sub record_update_extra_fields {
     };
 }
 
+=head2 inform_about_shortlisted
+
+We send email and text to inspector when report (re)allocated to them.
+
+=cut
+
+sub inform_about_shortlisted {
+    my ( $self, $user, $report ) = @_;
+
+    # XXX In future we may want to do some extra filtering here, to only send
+    # to users with specific roles or
+    # for reports with specific categories
+
+    my $email = $user->email;
+    my $phone = $user->phone;
+
+    if ($email) {
+        my $h = {
+            cobrand => $self,
```

Comment:
I think I'd create a problem_url variable here, then don't need to call base_url_for_report etc in the new templates?